modules/openapi-generator/src/test/java/org/openapitools/codegen/TestUtils.java

```diff
@@ -83,31 +88,32 @@ public static OpenAPI createOpenAPIWithOneSchema(String name, Schema schema) {
         return openAPI;
     }
 
+    /**
+     * Extract file from {@link MockDefaultGenerator}
+     *
+     * @param generator Generator
+     * @param root root path
+     * @param filename filename under root
+     *
+     * @return a {@link WrittenTemplateBasedFile}
+     * @deprecated Since 5.0. Please avoid this method and usage of {@link MockDefaultGenerator}, prefer {@link DefaultGenerator#DefaultGenerator(Boolean)} with dryRun=true.
```

Comment:
`MockDefaultGenerator` duplicates all written bytes. `getTemplateBasedFile` is used to "inspect" data passed to templates, but this should be tested differently in the future.